src/pages/workspaces/workspace/Files.ts

```diff
@@ -17,11 +18,17 @@ export const Files = _.flow(
     topBarContent: null
   })
 )(({ workspace }, _ref) => {
-  const { bucketName, googleProject } = workspace.workspace
-
   const fileBrowserProvider = useMemo(
-    () => GCSFileBrowserProvider({ bucket: bucketName, project: googleProject }),
-    [bucketName, googleProject]
+    () => {
+      if (workspace.azureContext) {
```

Comment:
Should also be able to use `workspace.cloudPlatform` now.

Reply:
Updated this to use `isAzureWorkspace` from workspace-utils. That function does currently use `azureContext`,b ut we can change that later. https://github.com/DataBiosphere/terra-ui/pull/3580/commits/422f53f2ea59dfe8d86f3e4efdd75e5992912aab.